src/full/Agda/TypeChecking/Conversion.hs

```diff
@@ -111,6 +112,14 @@ intersectVars = zipWithM areVars where
     areVars (Apply (Arg _ (Var n []))) (Apply (Arg _ (Var m []))) = Just $ n /= m -- prune different vars
     areVars _ _                                   = Nothing
 
+-- | Perform an action if two arguments are different as pointers, or report performance statistics otherwise.
+-- Use with care, see the document for @unsafeComparePointers@
```

Comment:
If you use `'f'` instead of `@f@`, then you get a link to `f` (if `f` is in scope).

Reply:
And please state clearly that `guardPointerEquality x y s m` should have the same semantics as `m` (for some notion of "the same").

Reply:
@nad is this resolved?